### ISSUE TITLE
[1.17.1] Crash / modloading error when loading mod on a server on BisectHosting.

### ISSUE THREAD

**Issue by Gbergz** [PrinceOfAmber/Cyclic#1971]
## Strange one I know, here goes:

**Minecraft Version:** 1.17.1

**Forge Version:** 37.0.109

**Mod Version:** 1.6.0, 1.6.1 & 1.6.2 (all tested)

**Single Player or Server:** Server

**Describe problem (what you were doing; what happened; what should have happened):**
Loading the mod onto a server hosted on BisectHosting server crashes / gets modloading error.
Locally it's totally fine. I have no idea why it only happens on hosted servers for example BisectHosting.

**Log file link:**
https://gist.github.com/Gbergz/52386a64ded47b11f3706cf0801de3a9

**Crash Report File**
https://gist.github.com/Gbergz/a00d1ded55ceb1c0b111888478eed0b2

**-** Gbergz

**Comment by Gbergz** [comment]
Any clues, ideas? 
Can't boot up the public server for my modpack because of this particular issue.

MmmMmmMmmMmm has a similar issue found here with more context:
https://www.curseforge.com/minecraft/mc-mods/mmmmmmmmmmmm/issues/6

**Comment by Lothrazar** [comment]
1.6.2 fixed a server start up with bugged recipes. 

For your log, do you have a cyclic-client.toml on the server? can you try deleting that https://www.curseforge.com/minecraft/mc-mods/cyclic/files/3528855

**Comment by Gbergz** [comment]
Yeah, using 1.6.2 now, with `cyclic-client.toml` deleted. Crash still happens.
https://gist.github.com/Gbergz/b120bc8e9d185bfce8fc2b45427edf1d

**Comment by Gbergz** [comment]
Same on 1.6.3

**Comment by Lothrazar** [comment]
Minecraft v1.17.1 is no longer supported, but this issue may have been fixed in Cyclic-1.17.1-1.6.5 or 1.6.4, which is newer than the latest mentioned viersion.   if not its for sure fixed in minecraft 1.18.2+
